mockld/polling_service.go

```diff
@@ -36,7 +36,7 @@ func NewPollingService(
 		router.HandleFunc("/sdk/flags/latest-all", pollHandler).Methods("GET")
 	case MobileSDK:
 		router.HandleFunc("/msdk/evalx/users/{user}", pollHandler).Methods("GET")
-		router.HandleFunc("/msdx/evalx/user", pollHandler).Methods("REPORT")
+		router.HandleFunc("/msdk/evalx/user", pollHandler).Methods("REPORT")
```

Comment:
Whoops

Reply:
This didn't cause a failure because there isn't currently a test that uses polling mode for mobile.